glean-core/metrics.yaml

```diff
@@ -498,8 +500,6 @@ glean.error:
       Only sent if the buffer ever overflows.
 
       In Version 0 this reported the total number of tasks enqueued.
-    unit:
```

Comment:
o_O. Where did this come from?  Ahh, cool! Even though `unit` has not meaning here, it _still shows up in the Glean dictionary_. Maybe this would be okay to leave as I kinda like the explicit documentation this metadata provides (though the description should do that, too) 🤔 

No strong opinions on this, happy either way.

Reply:
I was equally surprised!
We just disallowed this in glean_parser. So we would need to retract that decision, which would require a bug, a revert and a new release.